cargo-fix: add page (#10774)
tldr-pages/tldr: pages/common/cargo-fix.md

# cargo fix

> > Automatically fix lint warnings reported by rustc.

Reviewer: ```suggestion
> Automatically fix lint warnings reported by `rustc`.
```

> More information: <https://doc.rust-lang.org/cargo/commands/cargo-fix.html>.

- Fix code even if it already has compiler errors:

`cargo fix --broken-code`

- Fix code even if the working directory has changes:

`cargo fix --allow-dirty`

- Fix the package’s library:

`cargo fix --lib`

- Fix the specified integration test:

> `cargo fix --test`

Reviewer: ```suggestion
`cargo fix --test {{name}}`
```

- Fix all members in the workspace:

`cargo fix --workspace`

> - Directory for all generated artifacts and intermediate files:
> 
> `cargo fix --target-dir`

Reviewer: ```suggestion
- Set the directory for all generated artifacts and intermediate files:

`cargo fix --target-dir {{path/to/directory}}`
```

- Restrict Cargo from access to network for any reason:

`cargo fix --offline`

> - Number of parallel jobs to run:
> 
> `cargo fix --jobs`

Reviewer: ```suggestion
- Run `n` jobs in parallel (default: number of logical CPUs):

`cargo fix --jobs {{n}}`
```
